src/components/TodoList.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <>
```

Comment:
```suggestion
    <section className="todoapp__main" data-cy="TodoList">
```
You can move section here so you don't have to use react fragment